release-02/refactor/Gemfile

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+gem 'roda'
```

Comment:
I think we have to add `sqlite3` as a dependency, I got errors running `rackup` before I installed it.

Reply:
You are absolutely right. I wonder why it is not in there.

Reply:
Has been added.